config/style.yml

```diff
@@ -147,41 +141,17 @@ Style/GuardClause:
   Description: 'Check for conditionals that can be replaced with guard clauses'
   Enabled: false
 
-# Introduced in: c69a7eb3af955d6c4c0cf0c3cec8e9f330c74429
-# TODO: unclear why this is here!
-Style/MethodDefParentheses:
-  Description: >-
-    Checks if the method definitions have or don't have
-    parentheses.
-  Enabled: false
-
-# Introduced in: c69a7eb3af955d6c4c0cf0c3cec8e9f330c74429
-# TODO: unclear why this is here!
-Style/NegatedIf:
-  Description: >-
-    Favor unless over if for negative conditions
-    (or control flow or).
-  Enabled: false
-
 # Introduced in: c69a7eb3af955d6c4c0cf0c3cec8e9f330c74429
 # TODO: unclear why this is here!
 Style/Next:
   Description: 'Use `next` to skip iteration instead of a condition at the end.'
   Enabled: false
 
-# Introduced in: c69a7eb3af955d6c4c0cf0c3cec8e9f330c74429
-# TODO: unclear why this is here!
+# We should avoid Cops that are based on heuristics. In this
+# case, it's unclear why having more than 5 digits should change
+# the way we type (why not 4?). If there's a readability issue
+# with a particular number, we can always fix that specifically.
 Style/NumericLiterals:
```

Comment:
I think this one might be less controversial/arbitrary than it sounds. 

With it enabled we get 100_00 and 10000 as requiring needing to be changed to 10_000. I think the number of characters (5) seems reasonable as it's once you're past the 9999 region that it can be difficult for a brain to parse the number.

Reply:
Sure, I agree it's unlikely someone would be motivated enough to subvert this Cop.

Reply:
Updated.